ee/fleet-agent-downloader/api/controllers/entrance/signup-okta-user-or-redirect.js

```
@@ -15,12 +15,15 @@ module.exports = {
 
 
   fn: async function () {
-
+    // If the okta sso hook is not configured, redirect the user to the login page.
+    if(!sails.config.custom.oktaClientSecret) {
```

ESLint

[error] Insert `·`

(prettier/prettier)